resources/client/cl_main.ts

```diff
@@ -1,17 +1,14 @@
 import { sendMessage } from '../utils/messages';
 import { PhoneEvents } from '../../typings/phone';
 import { TwitterEvents } from '../../typings/twitter';
-import { MessageEvents } from '../../typings/messages';
 import { config } from './client';
 import { animationService } from './animations/animation.controller';
 import { RegisterNuiCB } from './cl_utils';
 
 // All main globals that are set and used across files
 (global as any).isPhoneOpen = false;
 (global as any).isPhoneDisabled = false;
-
-// Set a global function to check if the player has been loaded
-(global as any).isPlayerLoaded = config.general.enableMultiChar ? false : true;
+(global as any).isPlayerLoaded = false;
```

Comment:
With the amount of flags we have now globally, lets actually be type safe and extend our global type to include our custom defined vars.

**global.d.ts** in `client/types/global.d.ts`

```ts
declare global {
  namespace NodeJS {
    interface Global {
      isPhoneOpen: boolean;
      isPhoneDisabled: boolean ;
    // ... etc
    }
  }
}
export {};
```